assets/csidriveroperators/manila/01_namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Do we want manila in this PR?

Is it possible to start with EBS and add manila in a subsequent PR?

Reply:
I wanted to have manila in the code so I know the structures work. I'll remove manila.